### ISSUE TITLE
ansible-galaxy fails with "ERROR! Unsupported Galaxy server API version: /api/v1/\n"

### ISSUE THREAD

**Issue by bitmensch** [ansible/ansible#13493]
##### Issue Type:

Bug Report
##### Ansible Version:

ansible@devel
##### Ansible Configuration:

no changes
##### Environment:

Mac OS X 10.11.2
##### Summary:

Installing ansible roles from requirements file fails with "ERROR! Unsupported Galaxy server API version: /api/v1/\n". This has been definitely working before.

At a first glance ansible/lib/ansible/galaxy/api.py:GalaxyAPI SUPPORTED_VERSIONS list seems to contain something like "v1", but URL paths are tested against it, e.g. "/api/v1\n".
##### Steps To Reproduce:

Install role via requirements file with ansible-galaxy.
You can see the error in action at https://travis-ci.org/benjixx/goodplay/builds/95871329
##### Expected Results:

Install should succeed.
##### Actual Results:

"ERROR! Unsupported Galaxy server API version: /api/v1/\n"

**Comment by jimi-c** [comment]
cc @chouseknecht

**Comment by spk** [comment]
Same problem with `ansible@stable-2.0`

**Comment by bitmensch** [comment]
It is working now again with `ansible@devel`.
At a first glance I cannot see any fix for this in the ansible code itself, but it probably has been fixed on the ansible-galaxy server side.

**Comment by jimi-c** [comment]
@benjixx yes I relayed this to @chouseknecht, so maybe it was a server-side issue.

If that is the case, we can go ahead and close this.

**Comment by bitmensch** [comment]
Yes, so would be good if @chouseknecht could verify this. Afterwards it can be closed.

**Comment by chouseknecht** [comment]
I verified, just now, that installing via requirements file works in 1.9.4 stable:

```
ansible-galaxy install -s galaxy-qa.ansible.com --force -r requirements
- downloading role 'ansible-grub', owned by chouseknecht
- downloading role from https://github.com/chouseknecht/ansible-grub/archive/v0.1.1.tar.gz
- extracting chouseknecht.ansible-grub to /Users/chouseknecht/projects/roles/chouseknecht.ansible-grub
- chouseknecht.ansible-grub was installed successfully
```

And in 2.0 devel with the galaxy-2.0-update PR applied:

```
ansible-galaxy install -s https://galaxy-qa.ansible.com --force -r requirements
[DEPRECATION WARNING]: going forward only the yaml format will be supported. This feature will be removed in a future
release. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
- downloading role 'ansible-grub', owned by chouseknecht
- downloading role from https://github.com/chouseknecht/ansible-grub/archive/v0.1.1.tar.gz
- extracting chouseknecht.ansible-grub to /Users/chouseknecht/projects/roles/chouseknecht.ansible-grub
- chouseknecht.ansible-grub was installed successfully
```

It may be that I fixed something in galaxy-2.0-update.

**Comment by jimi-c** [comment]
@chouseknecht thanks! Closing this now.
